modules/jdbc-test/src/test/java/org/testcontainers/jdbc/JDBCDriverAliasPropertiesTest.java

```diff
@@ -0,0 +1,97 @@
+package org.testcontainers.jdbc;
+
+import static java.util.Arrays.asList;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertTrue;
+
+import java.sql.SQLException;
+import java.util.EnumSet;
+import java.util.Properties;
+
+import org.apache.commons.dbutils.QueryRunner;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+
+@RunWith(Parameterized.class)
+public class JDBCDriverAliasPropertiesTest {
+
+    private enum Options {
+        ScriptedSchema,
+        CharacterSet,
+        CustomIniFile,
+        JDBCParams,
+        PmdKnownBroken
+    }
+
+    @Parameter
+    public String jdbcUrl;
+    @Parameter(1)
+    public EnumSet<Options> options;
+
+    @Parameterized.Parameters(name = "{index} - {0}")
+    public static Iterable<Object[]> data() {
+        return asList(
+            new Object[][]{
+                {"jdbc:tc:mysqlserver://hostname:hostport;databaseName=databasename", EnumSet.noneOf(Options.class)}                
+            });
+    }
+    
+    @Test
+    public void shouldInstanciateContainersAccordingToPropertiesWithDefaultImageTag() throws SQLException {
+    	Properties p = new Properties();
+    	p.put("jdbc.alias.mysqlserver.type", "sqlserver");
+    	p.put("jdbc.alias.mysqlserver.image", "mcr.microsoft.com/mssql/server");
+    	ContainerDatabaseDriver.clearJBDCAliasProperties();
```

Comment:
this should use the configuration mock, see other configuration-sensitive tests (e.g. tests for the reusable mode)